src/main/resources/assets/admin/common/styles/api/util/htmlarea/modal-dialog.less

```diff
@@ -310,10 +312,16 @@
 
     .scroll-up-div {
       top: 0;
+      .arrow {
+        background-image: url('../../../../images/arrow_up.png');
+      }
     }
 
     .scroll-down-div {
       bottom: 0;
+      .arrow {
+        background-image: url('../../../../images/arrow_down.png');
+      }
     }
 
```

Comment:
We shouldn't need ``.arrow``, ``.scroll-up-div`` and ``.scroll-down-div`` in ``modal-dialog.less``. The arrows are specific for the Insert Image dialog, so these classes should only be in ``image-dialog.less``